roles/galaxy.movedata/tasks/main.yml

```diff
@@ -1,5 +1,32 @@
-- include: galaxy_data.yml
-  when: check_galaxy_data
+- set_fact:
```

Comment:
How about a simpler mechanism:
We define a variable export_dir. This is the place where users mount persistent data.
If it does not exist, we do not persist. If it exists, but it is empty, we move the data/database/config and place symlinks to the new location. 
If it exists and is not empty, we do nothing (Well, maybe we check for correct permissions).
We tag this task and run a local round of ansible with only this task before starting supervisor.